lib/cached_network_image.dart

```diff
@@ -170,11 +270,11 @@ class CacheManager {
     var response;
     try {
       response = await http.get(url, headers: headers);
-    }catch(e){}
+    } catch (e) {}
```

Comment:
1. what is the default timeout on `.get`
2. how we can let developers know that `get` failed with certain error.
3. how developers can configure/get this error and decide what to do (present another image?)

Reply:
Didn't think about it yet. I do agree that it is a good idea improve the error handling.